go/connectors-client.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2024 Coralogix Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cxsdk
+
+import (
+	"context"
+
+	connectores "github.com/coralogix/coralogix-management-sdk/go/internal/coralogixapis/notification_center/connectors/v1"
+)
+
+// CreateConnectorRequest represents a request to create a connector.
+type CreateConnectorRequest = connectores.CreateConnectorRequest
+
+// CreateConnectorResponse represents a response to create a connector.
+type CreateConnectorResponse = connectores.CreateConnectorResponse
+
+// ReplaceConnectorRequest represents a request to replace a connector.
+type ReplaceConnectorRequest = connectores.ReplaceConnectorRequest
+
+// ReplaceConnectorResponse represents a response to replace a connector.
+type ReplaceConnectorResponse = connectores.ReplaceConnectorResponse
+
+// DeleteConnectorRequest represents a request to delete a connector.
+type DeleteConnectorRequest = connectores.DeleteConnectorRequest
+
+// DeleteConnectorResponse represents a response to delete a connector.
+type DeleteConnectorResponse = connectores.DeleteConnectorResponse
+
+// GetConnectorRequest represents a request to get a connector.
+type GetConnectorRequest = connectores.GetConnectorRequest
+
+// GetConnectorResponse represents a response to get a connector.
+type GetConnectorResponse = connectores.GetConnectorResponse
+
+// ListConnectorsRequest represents a request to list connectors.
+type ListConnectorsRequest = connectores.ListConnectorsRequest
+
+// ListConnectorsResponse represents a response to list connectors.
+type ListConnectorsResponse = connectores.ListConnectorsResponse
+
+// BatchGetConnectorsRequest represents a request to batch get connectors.
+type BatchGetConnectorsRequest = connectores.BatchGetConnectorsRequest
```

Comment:
for those bulk/batch requests, we already have some APIs exposing something similar, can you make sure they all use the same noun to describe it? so it's either batch or bulk across the SDK

Reply:
maybe I'm wrong, but I saw clients with bulk and others with batch, can you please instruct me which one to use?